functions/displayBlock.js

```diff
@@ -43,7 +43,7 @@ export default function displayBlock(block, index) {
     case 'core/spacer':
       return <Blocks.BlockSpacer {...attributes} key={index} />
     default:
-      return <pre key={index}>{JSON.stringify(attributes, null, 2)}</pre>
+      return <pre key={index}>{JSON.stringify(block, null, 2)}</pre>
```

Comment:
This apparently slipped in with a separate PR, but can you align the param and return comments in the function JSDoc?